cmd/cli/plugin/managementcluster/post_install_hook.go

```diff
@@ -3,12 +3,19 @@
 
 package main
 
+import (
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+
+	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/config"
+)
+
 func postInstallHook() error {
+	// Configure default feature-flags for management-cluster plugin
```

Comment:
Any thoughts on how to deal with situations where the defaults need to be repopulated for any reason after a plugin is installed?
e.g config file got nuked accidentally and though we could once use "tanzu config init" to re-create much of what is lost, this is not the case anymore.